hs628_tools/Visualization.py

```diff
@@ -0,0 +1,294 @@
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn as sns
+import pandas as pd
+import sys 
+import plotly.plotly as py
+import plotly.graph_objs as go
+#plotly.tools.set_credentials_file(username='hhowell', api_key='BRP1aHhfTHwkqnvalVQL')
```

Comment:
The substance of this commit, and the commit message immediately preceding this commit, suggest that this commit represents adding Haley's code; if that is the case then the commit message here should have made it clear.  Even better would be if it were a merge commit, such as would result if one of you had submitted a pull request to the other one's feature branch on their fork, and that PR was then merged into that feature branch (which is what I thought we had talked about), and then that feature branch was the basis for a PR into the main repository.  That way, that feature branch would carry all of the relevant git history for both authors along with it, instead of a bunch of it being collapsed into a single commit by just the one author.  

Since I'm already a lot of the way through commenting this amalgamated file, for the remaining cleanup work, which you'll both want to do respective parts of, it would probably be best for @haleyhowe  to branch off of @NikitaThomas feature branch that is the basis for this PR, then for each of you to make your respective changes, then for Haley to submit a PR to Nakita's feature branch, which Nakita would then merge into her branch to have those changes become part of this PR.  It would probably be worthwhile to step through that in class, but if we don't have time in class then we should do it after or schedule time over zoom.

Reply:
Hmm, now I see, though, that Haley's final version looks like it has a couple more commits cleaning up a couple of the issues with this final version.  But it looks like her final version also has some other stray code in it that's been cleaned up here, so it's not immediatlely clear which one would be the better basis for final cleanup, you guys choose.